codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerServiceGeneratorV2.kt

```diff
@@ -205,6 +207,13 @@ class ServerServiceGeneratorV2(
             }
         }
 
+        var missingOperationsUnusedMut = ""
+        var unusedExpected = ""
+        if (operations.isEmpty()) {
+            missingOperationsUnusedMut = "##[allow(unused_mut)]"
+            unusedExpected = "##[allow(unused_variables)]"
+        }
+
```

Comment:
I think you can use `Attribute.AllowUnusedMut/AllowUnusedVariables.render(writer)` for these.

Reply:
https://github.com/awslabs/smithy-rs/pull/2351/commits/e0a4d8316c92fa78d12c420ae7f58d89255585e1